src/main/scala/gemmini/AccumulatorMem.scala

```diff
@@ -291,8 +294,15 @@ class AccumulatorMem[T <: Data, U <: Data](
     }
   }
 
+
   val q = Module(new Queue(new AccumulatorReadResp(t, scale_t, log2Ceil(t.head.head.getWidth)),  1, true, true))
   q.io.enq.bits.data := rdata_for_read_resp
+
+  if (is_dummy) {
+    rdata_for_read_resp := DontCare
+    rdata_for_adder := DontCare
```

Comment:
Is it possible that Chisel could replace the `DontCare` with the output of the SRAM? It might be safer to set these to `0.U` instead.

Reply:
Firrtl treats DontCare as 0.U. The use of DontCare here is safer if the type for rdata ever changes, it avoids some code churn